ReSharper.FSharp/src/FSharp/FSharp.Psi/src/FSharp.psi

```diff
@@ -1925,6 +1926,17 @@ orTypeUsage:
   OR
   typeUsage;
 
+intersectionTypeUsage:
+  (
+    typeParameterDeclaration<TYPE_PARAMETER, TypeParameter>
```

Comment:
Is it always a declaration, and not a type usage, though? I'd probably expect the declarations to only be inside explicit type param decl list.

Reply:
there should be `typeReferenceName`